benchmarks/multi-turn-chat-go/example/pod.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: bench
+spec:
+  restartPolicy: Never
+  containers:
+    - name: bench
+      image: us-central1-docker.pkg.dev/substratus-dev/default/benchmark-multi-turn-chat-go
+      args: ["--config=/config/config.json", "--threads=./data/tiny.json"]
```

Comment:
I personally prefer passing the config as individual CLI args. Removes the need to configure volume, volumeMount and configmaps.

Reply:
I'll add that as an option